### PR TITLE
Optimize code for DuUIE Baseline

### PR DESCRIPTION
**Pull request by luyaojie**
### PR types

Performance optimization

### PR changes

Models 

### Description

* Remove `DynamicMultiTaskSSIGenerator` , using DynamicSSIGenerator for all Data Collator.
* Reduce GPU Memory to 50%. Rename `negative_sampler` to `ssi_generator`. Rewrite  `__call__` for each Data Collator. 
* Read training instances from multi-task train config and add negative instances for DuIE SPO datasets.
* Add more comments and rename variables in seq2struct/utils.py evaluation/seq2record.py for better understanding.
* Remove unused arguments and `tqdm` code in run_seq2struct.py, add grad_clip, and change the default value of some arguments as README.md
* Change BaseStructureMarker as data class.

### CODE REVIEW

File: examples/information_extraction/DuUIE/uie/seq2struct/utils.py [hunk 1]
```diff
@@ -114,66 +121,147 @@ def set_logger(args):
...
-    logger.info(f'Load data from {train_filename} ...')
-    schema = RecordSchema.read_from_file(args.record_schema)
+def read_training_instance_based_config(tokenizer,
+                                        config_file: str,
+                                        max_source_length: int,
+                                        negative_keep: float=1.0):
+    """Read training instances based on config_file
+
+    Args:
+        config_file (str): Config filename
```
Comment (linjieccc) [inline examples/information_extraction/DuUIE/uie/seq2struct/utils.py:198]: 注释与函数参数没有对应

File: examples/information_extraction/DuUIE/uie/seq2struct/utils.py [hunk 2]
```diff
@@ -274,18 +351,29 @@ def get_eval_dataloader(model, tokenizer, eval_filename, record_schema, args):
 
 
 def load_eval_tasks(model, tokenizer, args):
+    """ Load evaluation tasks
+
+    Args:
+        model (_type_): _description_
```
Comment (linjieccc) [inline examples/information_extraction/DuUIE/uie/seq2struct/utils.py]: 辛苦补充下这里的注释

Comment (linjieccc) [inline examples/information_extraction/DuUIE/uie/seq2struct/utils.py:357]: 辛苦补充下这里的注释

File: examples/information_extraction/DuUIE/uie/seq2struct/utils.py (hunk 1 above)
Comment (luyaojie) [reply examples/information_extraction/DuUIE/uie/seq2struct/utils.py:198]: 已经添加

File: examples/information_extraction/DuUIE/uie/seq2struct/utils.py (hunk 2 above)
Comment (luyaojie) [reply examples/information_extraction/DuUIE/uie/seq2struct/utils.py]: 这里也已经补充

**Review by linjieccc — APPROVED** [review]
LGTM

**Pull request merged**